blog/_posts/2021-10-25-tilt-news-october-2021.md

```diff
@@ -0,0 +1,97 @@
+
+We've written some posts to help people think through this:
+
+- [Improving Docker performance](https://blog.tilt.dev/2021/09/13/docker-does-not-mean-slow.html)
+- [Swapping Rancher Desktop for Docker Desktop](https://blog.tilt.dev/2021/09/07/rancher-desktop.html)
+
+## Kubernetes Cluster API
+
+In other Kubernetes news, Cluster API is now officially production-ready!
+
+Cluster API is a Kubernetes project that enables declarative management for Kubernetes, using APIs to easily create, configure, and update clusters. It is an end-to-end approach that can simplify the repetitive tasks of the Kubernetes lifecycle, while maintaining consistency and repeatability across a unified infrastructure.
+
+They are also the biggest open-source project that uses Tilt. Check out [their developer guide](https://cluster-api.sigs.k8s.io/developer/tilt.html) to see how they are leveraging Tilt for rapid iterative development.
+
+## Tilt News
+
+We were thrilled to see Tilt mentioned in three blogposts:
+
+Kevin Lindsay, Full Stack Web Developer at Surge and OpenFaas community member, writes about using Tilt to bring smart rebuilds and live updates to [OpenFaaS functions](https://www.openfaas.com/blog/tilt/).
+
+Archers.ai’s mission is to help its users track cloud resources for both cost-optimization and tax purposes. They care a lot about the ability to track the cost of dev environments, and they see tools like Tilt as essential for helping them do that. In this blog post, Jason Burt, Product at Archera.ai explains how they are using Tilt to [accelerate their product feedback loop](https://archera.ai/blog/how-we-used-tilt/).
+
+Evgeny Khabarov, Consultant and Gopher, wrote this [four part series](https://dev.ms/2021/10/envoy-as-an-api-gateway-part-iii/) on building a RESTful API with Envoy. Tilt is featured as tool of choice for developing on a local Kubernetes cluster with ease. 
+
+### Guinea pigs wanted!
+
+We're always trying to design and improve our product as close to the community's needs as possible. Whether you've been a long-time fan of Tilt or have never used it before, we'd love to invite you for a user-research session. You might be able to catch a glimpse of upcoming features!
+A session is typically between 45 and 60 minutes long and you will receive a giftcard of around $50 USD (or local currency equivalent) as a thank you.
+
+Sound right up your alley? Then head on over to our [intake form](https://forms.gle/gecjWQ6ErHGfJNm66).
+
+### Enable & Disable Services
+
+And last but not least, we're excited to let you know that we are currently working on a feature to easily disable and enable specific services on `tilt up` and at runtime. While we are cautious to make any definite statements on release dates, we'd still like to give you a little sneak peek to get y'all hyped!
```

Comment:
As a general rule, I think it's better to describe the problem rather than the specific feature. This doc might help:
https://github.com/tilt-dev/tilt.specs/blob/master/enable_disable_resources_user_story.md